public/pages/AnomalyCharts/containers/AnomaliesChart.tsx

```diff
@@ -266,6 +271,12 @@ export const AnomaliesChart = React.memo((props: AnomaliesChartProps) => {
                         onDisplayOptionChanged={props.onDisplayOptionChanged}
                         heatmapDisplayOption={props.heatmapDisplayOption}
                         isNotSample={props.isNotSample}
+                        // Category fields in HC results are always sorted alphabetically. To make all chart
```

Comment:
Do we reply on the order of category fields to show HC detector anomaly result?

Reply:
Yes, in the y-axis labels on the heatmap chart. From @kaituo, the results are sorted alphabetically (also see [here](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/org/opensearch/ad/model/Entity.java#L126)). To be consistent on the frontend, we sort the same way.

As an example, given some detector with category fields `[a, b]`, the heatmap results will be shown with labels `<a-value>, <b-value>`. In the chart header, we list out the category field names via the header `Sort by: a, b`. If we don't sort the category field, then it is possible the listed fields would be in a different order than the shown results. For example, if the user saves a detector with category fields `[b, a]`, then without sorting, the chart header would be `Sort by: b, a`, but the results would still be `a, b`, which could lead to confusion.